core/network/fabric-interop-cc/libs/assetexchange/assetSwapContracts.go

```diff
@@ -51,6 +51,8 @@ const (
 	assetKeyPrefix    = "AssetKey_"   // prefix for the map, asset-key --> asset-object
 	assetKeyDelimiter = "_"           // delimiter for the asset-key
 	contractIdPrefix  = "ContractId_" // prefix for the map, contractId --> asset-key
+	claimAssetKeyPrefix = "ClaimAssetKey_"
+	claimContractIdPrefix = "ClaimContractId_"
```

Comment:
Would it be useful to reconcile these keywords with those used for asset transfers in `libs/utils.go`? Later, we can then extract all constants into a single file.